fhir2dataset/graph_tools.py

```diff
@@ -23,12 +22,13 @@ def join_path(graph: nx.Graph) -> list:
 
     Returns:
         list -- List of tuples indicating the successive joints to be made
-    """
+    """  # noqa
```

Comment:
do we really have to have these `# noqa` statements everywhere ?

Reply:
Indeed I spent some time to find out what were the good practices concerning docstring and I did not find anything satisfactory/consensus. 
For lack of a clear answer, I had the impression that it was the easiest to do because in this configuration the lines of the documentation are not split in the middle (which might not be pleasant to read on IDEs for example)

Reply:
alright we'll clean that up later